Parse-Dashboard/index.js

```diff
@@ -6,31 +6,25 @@
  * the root directory of this source tree.
  */
 // Command line tool for npm start
+"use strict"
+const packageJson = require('package-json');
+const basicAuth = require('basic-auth');
+const path = require('path');
+const jsonFile = require('json-file-plus');
+const express = require('express');
 
-var DEFAULT_DASHBOARD_CONFIG = __dirname + '/parse-dashboard-config.json';
-
-var program = require("commander");
-program.option('--port [port]', "the port to run parse-dashboard");
-program.option('--config [config]', "the path to the configuration file");
-program.option('--allowInsecureHTTP [allowInsecureHTTP]', 'set that flag when parse server is behind an HTTPS load balancer/proxy');
+const program = require('commander');
+program.option('--appId [appId]', 'the app Id of the app you would like to manage.');
+program.option('--masterKey [masterKey]', 'the master key of the app you would like to manage.');
+program.option('--serverURL [serverURL]', 'the server url of the app you would like to manage.');
+program.option('--appName [appName]', 'the name of the app you would like to manage. Optional.');
+program.option('--config [config]', 'the path to the configuration file');
```

Comment:
The danger with non-named arguments is that it's easy to forget which ones exist and in what order, especially as the arguments list grows. I prefer named arguments for everything. This also would be easy to add later, but difficult to to remove later.